engine/src/function.cpp

```diff
@@ -958,12 +962,16 @@ QList<quint32> Function::components()
  * Flash
  *****************************************************************************/
 
-void Function::flash(MasterTimer *timer)
+void Function::flash(MasterTimer *timer, bool override, bool forceLTP)
 {
     Q_UNUSED(timer);
 
     if (m_flashing == false)
+    {
         emit flashing(m_id, true);
+        m_flashOverrides = override;
```

Comment:
I suspect some compilers will complain about this sooner or later, since it's a reserved keyword.
https://en.cppreference.com/w/cpp/language/override
Please rename